packages/oui-navbar/navbar.less

```diff
@@ -19,15 +19,6 @@
     #oui > .navbar-link();
     #oui > .navbar-link_variant();
 
-    &:hover,
```

Comment:
It was overridden by &-menu &-link so I deleted it @AxelPeter.
ok?

Reply:
Yes, but it can be useful for the base links in the main navbar (without `&__primary` or `&__secondary`).
For this case, we can use these variables instead:

```
&:hover,
&:focus,
&_active {
  #oui > .navbar-link_variant(
    @background-color: @oui-navbar-background-color_active,
    @font-color: @oui-navbar-font-color
  );
}
```